.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```

Comment:
if possible I should assign labels for bug

Reply:
Agree